extensions/wikia/AssetsManager/config.php

```diff
@@ -2088,14 +2088,25 @@
 	'skin' => [ 'oasis' ],
 	'assets' => [
 		'//resources/wikia/modules/scrollToLink.js',
+		'//resources/wikia/libraries/headroom/headroom.js',
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemTracking.js',
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemDropdowns.js',
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationScrollToLink.js',
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemAuthenticationMenu.js',
 		'//extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearch.js',
+		'//extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationHeadroom.js',
 	],
 ];
 
+$config['headroom_css'] = [
+	'type' => AssetsManager::TYPE_CSS,
+	'skin' => [ 'oasis' ],
+	'assets' => [
+		'//resources/wikia/libraries/headroom/headroom.css',
```

Comment:
Then that person should extract it. If it is used in design-system only i would put it there